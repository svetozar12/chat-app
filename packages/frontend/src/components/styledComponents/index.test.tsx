/** @jsx jsx */
import renderer from "react-test-renderer";
import * as exports from "./index";
import { jsx } from "@emotion/react";
import { createSerializer } from "@emotion/jest";

expect.addSnapshotSerializer(createSerializer());

test("Renders Label_container", () => {
  expect(renderer.create(<exports.Label_container>hi</exports.Label_container>).toJSON()).toMatchSnapshot();
});
test("Renders Form_header", () => {
  expect(renderer.create(<exports.Form_header>hi</exports.Form_header>).toJSON()).toMatchSnapshot();
});
test("Renders Form", () => {
  expect(renderer.create(<exports.Form>hi</exports.Form>).toJSON()).toMatchSnapshot();
});
test("Renders Button", () => {
  expect(renderer.create(<exports.Button>hi</exports.Button>).toJSON()).toMatchSnapshot();
});
test("Renders Input", () => {
  expect(renderer.create(<exports.Input>hi</exports.Input>).toJSON()).toMatchSnapshot();
});
test("Renders Label_button", () => {
  expect(renderer.create(<exports.Label_button>hi</exports.Label_button>).toJSON()).toMatchSnapshot();
});
