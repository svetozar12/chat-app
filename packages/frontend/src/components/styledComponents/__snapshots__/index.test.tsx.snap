// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Renders Button 1`] = `
.emotion-0 {
  margin-top: 1rem;
  border-radius: 5px;
  width: 60%;
  background-color: var(--button-blue);
  color: var(--main-white);
  border: 1px solid var(--input-border-color);
  padding: 1rem;
  cursor: pointer;
  -webkit-transition: 0.2s;
  transition: 0.2s;
  font-weight: bold;
  font-size: 1rem;
}

.emotion-0:hover {
  opacity: 0.7;
  -webkit-transition: 0.2s;
  transition: 0.2s;
}

<button
  className="emotion-0"
>
  hi
</button>
`;

exports[`Renders Form 1`] = `
.emotion-0 {
  width: 40%;
  height: 60vh;
  padding: 3rem;
  border-radius: 5px;
  background: #fff;
  box-shadow: 0 2px 30px rgba(0, 0, 0, 0.1);
}

@media (max-width: 1010px) {
  .emotion-0 {
    padding: 0rem;
    width: 90%;
  }
}

<form
  className="emotion-0"
>
  hi
</form>
`;

exports[`Renders Form_header 1`] = `
.emotion-0 {
  padding: 1rem;
  height: auto;
  text-align: center;
  margin: 0;
  width: 40%;
  border-top-right-radius: 5px;
  border-top-left-radius: 5px;
  color: var(--main-white);
  background: var(--form-gray);
}

@media (max-width: 1010px) {
  .emotion-0 {
    width: 90%;
  }
}

<h1
  className="emotion-0"
>
  hi
</h1>
`;

exports[`Renders Input 1`] = `
.emotion-0 {
  width: 100%;
  height: 2rem;
  margin: 0.5rem 0;
  border: 1px solid var(--input-border-color);
  border-radius: 5px;
  -webkit-transition: 0.3s;
  transition: 0.3s;
  padding: 1.3rem 0.9rem;
}

<input
  className="emotion-0"
>
  hi
</input>
`;

exports[`Renders Label_button 1`] = `
.emotion-0 {
  margin: 0 2rem;
  border-radius: 5px;
  width: 4rem;
  text-align: center;
  padding: 0.5rem;
  color: var(--main-white);
  background: rgba(0, 105, 217, 0.5);
  cursor: pointer;
  display: block;
}

<label
  className="emotion-0"
>
  hi
</label>
`;

exports[`Renders Label_container 1`] = `
.emotion-0 {
  dispay: flex;
  -webkit-flex-direction: collumn;
  -ms-flex-direction: collumn;
  flex-direction: collumn;
  width: 60%;
}

<div
  className="emotion-0"
>
  hi
</div>
`;
